bundles/org.openhab.binding.tapocontrol/README.md

```diff
@@ -0,0 +1,97 @@
+# TapoControl Binding
+
+This binding adds support to control Tapo (Copyright © TP-Link Corporation Limited) Smart Home Devices from your local openHAB system.
+(Note: This is not an official Tapo/TP-link Binding!)
+
+## Supported Things
+
+The following Tapo-Devices are supported
+
+### P100 SmartPlug (WiFi)
+
+* Power On/Off
+* Wi-Fi signal (SignalStrenght)
+* On-Time (Time in seconds device is switched on)
+
+
+### L530_Series MultiColor SmartBulb (WiFi)
+
+* Light On/Off
+* Brightnes (Dimmer)  0-100 %
+* ColorTemperature (Number) 2500-6500 K
+* Color (Color)
+* Wi-Fi signal (SignalStrenght)
+* On-Time (Time in seconds device is switched on)
+
+## Prerequisites
+
+Before using Smart Plugs with openHAB the devices must be connected to the Wi-Fi network.
+This can be done using the Tapo provided mobile app.
+
+## Discovery
+
+Discovery is not supported at the moment. You need to know the IP-Adress of your device
+
+## Binding Configuration
+
+Binding needs your Tapo eMail and password to connect to the Tapo-Cloud.
+This is only used to create the handshake (cookie) to act with your devices.
+
+To enter your cloud details go to the bindings page, click the TapoControl binding and than configure.
+![Binding Config](doc/bindingConfig1.png)
+
+In the configuration page, enter your eMail and password.
+![Binding Config](doc/bindingConfig2.png)
```

Comment:
It's uncommon to integrate screenshots for the default UI as they will be outdated if the UI changes one day. To prevent adjusting all readmes, it's better to avoid screenshots where possible.